src/modules/rest/controllers/common/rest.common.queries.ts

```diff
@@ -0,0 +1,27 @@
+export const COMMON_IMPLICIT_QUERIES = [
+    {
+        name: 'world',
+        required: false,
+        type: Number,
```

Comment:
Should we be using the primitive "number" instead?

Reply:
Capitalised is the primitive when it's used in the object context, otherwise it's invalid and won't compile